src/bootstrap/config.rs

```diff
@@ -519,6 +521,7 @@ impl Config {
                 }
             })
             .unwrap_or_else(TomlConfig::default);
+        config.version = toml.version;
```

Comment:
This is appearing right after toml which I think should be separated.
```suggestion

        config.version = toml.version;
```